Lib/traceback.py

```diff
@@ -140,7 +140,7 @@ def _print_exception_bltin(exc, /):
 
 
 def format_exception(exc, /, value=_sentinel, tb=_sentinel, limit=None, \
-                     chain=True):
+                     chain=True, **kwargs):
```

Comment:
Why doesn't this work?
```suggestion
                     chain=True, colorize=False):
```
Adding `**kwargs` to the public signature would be unfortunate.

Could also make it a private parameter, if you don't want to document it:
```suggestion
                     chain=True, _colorize=False):
```

Reply:
> Why doesn't this work?

I don't want to be exposed in the signature even with `_colorize`